1904-11-04.xml

```diff
@@ -0,0 +1,2581 @@
+                        granted till the Sanitary Department has satisfied itself that the shop or
+                        place of sale is well-aired, dry, and clean, and displays the name of the
+                        person licensed in legible charaoters on its front. </p>
+                    <p>The first examination will be held on December 5, 1904.</p>
+                </div>
+                <div type="item">
+                    <head>TARDY TELEGRAMS.</head>
+                    <p>Our Tantah Correspondent writes:-I had occasion a few days ago to complain
+                        about the delay in transmission of telegrams between here and Alexandria.
+                        Nothing seems to have been done as yet to remedy this inconvenience. During
+                        the last few days messages have taken about two hours in transmission from
+                        Alexandria, and many are the persons who have received telegrams advising
+                        them of friends passing through Tantah after the train with their friends
+                        had left! This is a serious state of things which should not be allowed to
+                        continue. It is to be hoped that we shall soon have the telephone line
+                        working, and then the Telegraph Department will see their receipts diminish
+                        appreciably in consequence of their lax methods. </p>
+                </div>
+                <div type="item">
+                    <head>GUM ARABIC EXPORTS.</head>
+                    <p>The gum arabic exports during the month of September last amounted to 851,627
+                        kilos, valued at LE 19,162, as compared with 602,579 kilos. and LE 13,560 in
+                        the same month last year, showing an increase of 249,048 kilos. and LE
+                        5,602. The exports were divided among the different countries as follows-
+                        United Kingdom, LE. 9,177; Ger- mathy, LE 1,313; Austria-Hungary, LE 1,657;
+                        France, LE 695; Italy, LE 201; Turkey, LE 8; United States, LE 4,601;
+                        Belgium, LE. 1,143; other countries, LE. 367. </p>
+                </div>
+                <div type="item">
+                    <head>STEAMER MOVEMENTS</head>
+                    <p>The Moss liner Menes arrived at Liverpool yesterday afternoon. </p>
+                    <p>The Moss Hoer Seti sailed from Liverpool yrater day afternoon and is doe bere
+                        about the 17th lost, wish passengers, malls, and general cargo. The
+                        Khedivial Mall 8.8. Prince Abbas will leave Alexandria on Wedoraday for
+                        Pirnar, Beyrna, Mitylene, and Constantinople. </p>
+                </div>
+                <cb n="3"/>
+                <div type="section" feature="khedive">
```

Comment:
This is not a khedive section, because there is only one item about the khedive (the first one). Remove the section div.